portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,151 @@
+    this.toRotate = toRotate;
+    this.el = el;
+    this.loopNum = 0;
+    this.period = parseInt(period, 10) || 2000;
+    this.txt = '';
+    this.tick();
+    this.isDeleting = false;
+};
+TxtType.prototype.tick = function () {
+    var i = this.loopNum % this.toRotate.length;
+    var fullTxt = this.toRotate[i];
+
+    if (this.isDeleting) {
+        this.txt = fullTxt.substring(0, this.txt.length - 1);
+    } else {
+        this.txt = fullTxt.substring(0, this.txt.length + 1);
+    }
+
+    this.el.innerHTML = '<span class="wrap">' + this.txt + '</span>';
+
+    var that = this;
+    var delta = 200 - Math.random() * 100;
+
+    if (this.isDeleting) { delta /= 2; }
+
+    if (!this.isDeleting && this.txt === fullTxt) {
+        delta = this.period;
+        this.isDeleting = true;
+    } else if (this.isDeleting && this.txt === '') {
+        this.isDeleting = false;
+        this.loopNum++;
+        delta = 500;
+    }
+
+    setTimeout(function () {
+        that.tick();
+    }, delta);
+};
+window.onload = function () {
+    var elements = document.getElementsByClassName('typewrite');
+    for (var i = 0; i < elements.length; i++) {
+        var toRotate = elements[i].getAttribute('data-type');
+        var period = elements[i].getAttribute('data-period');
+        if (toRotate) {
+            new TxtType(elements[i], JSON.parse(toRotate), period);
+        }
+    }
+    // INJECT CSS
+    var css = document.createElement("style");
+    css.type = "text/css";
+    css.innerHTML = ".typewrite > .wrap { border-right: 0.08em solid black}";
+    document.body.appendChild(css);
+};
+
+
+// JS for creating/listing/deleting message elements in messages-list.html
+
+// Fetches tasks from the server and adds them to the DOM
+function loadMessages() {
+    fetch('/form-responses').then(response => response.json()).then((formResponses) => {
+        const taskListElement = document.getElementById('message-list');
+        formResponses.forEach((oneMessage) => {
+            taskListElement.appendChild(createTaskElement(oneMessage));
+        })
+    });
+}
+// Creates an element that represents a task, including its delete button
+function createTaskElement(oneMessage) {
+    const taskElement = document.createElement('li');
+    taskElement.className = 'message';
+
+    const titleElement = document.createElement('span');
+    titleElement.innerText = oneMessage.textValue;
+
+    const deleteButtonElement = document.createElement('button');
+    deleteButtonElement.innerText = 'Delete';
+    deleteButtonElement.className = 'fonts';
+    deleteButtonElement.addEventListener('click', () => {
+        deleteTask(oneMessage);
+
+        // Remove the task from the DOM.
+        taskElement.remove();
+    });
+
+    taskElement.appendChild(titleElement);
+    taskElement.appendChild(deleteButtonElement);
+    return taskElement;
 }
+// Tells the server to delete the task
+function deleteTask(oneMessage) {
```

Comment:
Try to keep your naming consistent. Not only do your servlet names vary (FormHandler, FormResponsesList, DeleteResponses) but also your JS methods (Task v Message - which also differs from the servlet naming convention of response)